src/models/elan1-r1/README.md

```diff
@@ -0,0 +1,29 @@
+# Eland 1U (elan1-r1)
+
+![Eland 1U](./img/elan1-r1-frontports.avif)
+
+The System76 Eland 1U is a rack-mount server with the following specifications:
+
+- Operating System
+    - Ubuntu Server 22.04 LTS (64-bit)
+- CPU options
+    - Supports AMD EPYC™ 7003 series processor family
+    - Single processor
+- Memory
+    - Up to 4096GB (16x256GB) ECC DDR4 @ 3200 MHz
+- Networking
+    - 2x 1GbE LAN ports (1 x Intel® I350-AM2)
+    - 1x 10/100/1000 management LAN
+- Expansion
+    - 1 x PCIe x16 slot (Gen4)
+- Ports
+    - Front: 2 x USB 3.0, 1 x Power button with LED, 1 x ID button with LED, 1 x NMI button, 1 x Reset button, 2 x LAN activity LEDs, 1 x HDD activity LED, 1 x System status LED
+    - Rear: 3 x USB 3.0, 1 x VGA, 1 x COM, 2 x RJ45, 1 x MLAN, 1 x ID button with LED
+- Power Supply
+    - 2 x 650W redundant PSUs, 80 PLUS Platinum
+- Dimensions
+    - 17.2” × 26” × 1.7” (43.8cm × 66.0cm × 43.5cm)
+- Weight
+    - 44 lbs (17.24 kg)
+- Model
+    - elan1-r1 - [Technical Documentation](./elan1-r1_manual.pdf)
```

Comment:
I didn't realize these were linked this way from the main sales pages, but for Tech Docs, I think Technical Documentation as the link text is a little redundant. The model number is also redundant here. I'd prefer we list it the model number of the PDF we're linking to, like:

```
- Chassis
    - [Gigabyte R152-Z3{1,2,3}](./chassis-manual.pdf)
```

Thoughts?

Reply:
On initial setup, we have `oem` and the serial as username/password. Can we get a way to show where the System76 serial sticker is placed on these servers? Do they have a plastic plate that slides out above the drive cages like on the `jacp13` servers?

Reply:
Sounds good to me! I'll make the change when I take another pass, if nobody gets to it before I do.